girder/cumulus/server/models/job.py

```diff
@@ -54,7 +54,7 @@ def create(self, user, job):
         job['userId'] = user['_id']
 
         self.save(job)
-        send_status_notification('job', job)
+        send_notification('job', 'status', job)
```

Comment:
As mentioned above, I would preserve the send_status_notification(...) signature